docs/content/stack/configuration-variables.md

```diff
@@ -68,6 +68,11 @@ Useful for CI environments that fake tty and thus "freeze" waiting for user inpu
 This is the domain name used for Docksal project URLs, i.e., `http://$PROJECT_NAME.$DOCKSAL_DNS_DOMAIN`. 
 Project named `myproject` will result in `http://myproject.docksal` URL by default.
 
+## DOCKSAL_DNS_IP
+
+Can be used to override IP binding for the docksal-dns system service. E.g., `DOCKSAL_DNS_IP=0.0.0.0`.
```

Comment:
Can be used to override IP binding for the docksal-dns system service, e.g., `DOCKSAL_DNS_IP=0.0.0.0`.